notification/src/interfaces/notificationService.ts

```diff
@@ -0,0 +1,6 @@
+import { NotificationRequest } from './notificationRequest'
```

Comment:
In my opinion, we should leave out the extra abstraction of the notification service interface and keep it simple. Unlike for the transformation service, I don't think there is a use case here for adding a different implementation of the notification service. What do you think @georg-schwarz?

Reply:
Doesn't hurt either.... But I agree, we could refactor a little... Also a directory called `interfaces` doesn't look appealing for me, rather structure after domain (code-executor, notification, etc..)
Makes also sense to have files for each different notification type.